src/main/java/codesquad/springcafe/controller/UserController.java

```diff
@@ -0,0 +1,42 @@
+package codesquad.springcafe.controller;
+
+import codesquad.springcafe.domain.User;
+import codesquad.springcafe.service.UserService;
+import org.springframework.ui.Model;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.ModelAttribute;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+
+@org.springframework.stereotype.Controller
+public class UserController {
+
+    private final UserService userService;
+
+    public UserController(UserService userService) {
+        this.userService = userService;
+    }
+
+    @GetMapping("/users/create")
+    public String join() {
+        return "/user/form";
+    }
+
+    @PostMapping("/users/create")
```

Comment:
```suggestion
    @PostMapping("/user")
```
요구사항과 다르긴 하지만 요렇게 하는게 더 좋을 것 같긴 해요 ㅎㅎ

Reply:
넵 우선순위 설정 확인해보겠습니다!
조언 감사드립니다 호눅스! 근데 한가지 궁금한 점이 있습니다. form을 불러오는 요청과 그 form에서 필드를 내보내는 요청의 url을 같게 했을 때, (ex - 회원 가입 form은 Get, "/users/create", 회원 가입 요청은 Post, "/users/create") Post의 최종 결과를 다시 form으로 돌려 보낼 수 있는 이점이 있다고 들었는데, 이 부분은 딱히 고려할 부분이 아닐까요?